.github/workflows/autoformat.yml

```diff
@@ -30,7 +30,6 @@ jobs:
             return context.payload.comment.body.trim() === 'autoformat';
       - name: Create status
         id: create-status
-        if: ${{ steps.check-comment.outputs.result == 'true' }}
```

Comment:
The output of this job (`check-comment`) depends on the output of this step. If this step is skipped, `fromJson` at line 18 throws an error: 

https://github.com/mlflow/mlflow/actions/runs/2955848476

```
The template is not valid. .github/workflows/autoformat.yml (Line: 18, Col: 19): Error reading JToken from JsonReader. Path '', line 0, position 0.,.github/workflows/autoformat.yml (Line: 19, Col: 12): Error reading JToken from JsonReader. Path '', line 0, position 0.,.github/workflows/autoformat.yml (Line: 20, Col: 12): Error reading JToken from JsonReader. Path '', line 0, position 0.,.github/workflows/autoformat.yml (Line: 21, Col: 20): Error reading JToken from JsonReader. Path '', line 0, position 0.
```